### PR TITLE
Sequence and Flow chart 

### PR DESCRIPTION
**Pull request by yusale9**
Created both a sequence chart and flow chart

### CODE REVIEW

File: student-work/2023/Deborah-Sorrels/daily-routine.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+J--Eat-->S[Go to the refrigerator];
+S-->T[Take out a yogurt cup, granola mix and the oat milk];
+T-->U[Open the yogurt and granola]
+U-->V[Pour some granola into the yogurt];
+V-->W[Stir and eat];
+W-->X[Throw away the yogurt cup and place the spoon near the sink];
+X-->Y[Get a cup and pour a glass of oat milk];
+Y-->Z[Drink and put glass by the sink];
+Z-->AA[Return the granola and the oat milk to the refrigerator];
```
Comment (lsnovich) [inline student-work/2023/Deborah-Sorrels/daily-routine.md:31]: Where you go to walk the dog after eating it is not clear how you got there.
The better thing to do is to have a decision
ie
Was the dog walked? yes - continue no - go grab a poop bag ....

Comment (lsnovich) [reply student-work/2023/Deborah-Sorrels/daily-routine.md:31]: The sequence diagram is great - please make the small change in the flow chart diagram.
Thanks

**Review by lsnovich — CHANGES_REQUESTED** [review]

File: student-work/2023/Deborah-Sorrels/daily-routine.md (hunk 1 above)
Comment (yusale9) [reply student-work/2023/Deborah-Sorrels/daily-routine.md:31]: Changes were made.  Thx

File: student-work/2023/Deborah-Sorrels/daily-routine.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+AA-->EE{Was the dog walked?};
+EE--Yes-->BB;
+EE--No-->K;
+
+
+
+
+R-->BB[Check messages];
+AA-->BB[Check messages];
```
Comment (Laura-Novich-OBW) [inline student-work/2023/Deborah-Sorrels/daily-routine.md:41]:
```suggestion
```

Comment (Laura-Novich-OBW) [reply student-work/2023/Deborah-Sorrels/daily-routine.md:41]: this can be removed

**Review by Laura-Novich-OBW — CHANGES_REQUESTED** [review]
one small change

**Review by Laura-Novich-OBW — APPROVED** [review]

**Comment by Laura-Novich-OBW** [comment]
nice going please submit to Google Classroom

**Pull request merged**